### PR TITLE
add acceptance tests

### PR DESCRIPTION
**Pull request by t-keazirian**
Added in Acceptance Tests from [https://github.com/8thlight/http_server_spec](url)

- Added `@wip` to all tests to skip for now

### CODE REVIEW

File: http_server_spec/features/01_getting_started/method_not_allowed.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@method-not-allowed @01-getting-started
+Feature: Method Not Allowed
+
+	@wip
+  Scenario: Finding GET for an endpoint with only HEAD
+    Given I make an GET request to "/head_request"
+    Then my response should have status code 405
+    And my response should have allowed headers of HEAD, OPTIONS
+    And my response should have an empty body
```
Comment (Aoxian) [inline http_server_spec/features/01_getting_started/method_not_allowed.feature:9]: What is up with the spacing on all these `@wip` additions? I see it on every single one.
```suggestion
  @wip
  Scenario: Finding GET for an endpoint with only HEAD
    Given I make an GET request to "/head_request"
    Then my response should have status code 405
    And my response should have allowed headers of HEAD, OPTIONS
    And my response should have an empty body
```

**Comment by Aoxian** [comment]
Is this PR for this ticket: [Add Acceptance Tests to CI Pipeline](https://trello.com/c/I3DgXkDd/34-add-acceptance-tests-to-ci-pipeline)?

I was expecting to see the CI updated to run the tests as well.

File: http_server_spec/features/01_getting_started/method_not_allowed.feature (hunk 1 above)
Comment (t-keazirian) [reply http_server_spec/features/01_getting_started/method_not_allowed.feature]: I'm not sure why it spaced it like that but I fixed it!

**Comment by Aoxian** [comment]
> > Add Acceptance Tests to CI Pipeline
> 
> No, this was simply to add the tests to my project. But this can be a part of that ticket and I can convert it to a draft. Or they can be two separate PRs.

Thanks; I think its easier as a review if its 1 PR per card. However, considering the shear number of files this adds, I actually appreciate you are handling it separately.  Its ok to include your thoughts (and a link to the card) in the description as well. It doesn't have to mimic the git commits.

**Review by Aoxian — APPROVED** [review]

**Pull request merged**